fixtures/setup.json

```diff
@@ -12,12 +12,12 @@
   ],
   "miners": [{
     "owner": 5,
-    "sealProofType": 3,
+    "sealProofType": 1,
```

Comment:
No, don't do this. The sectors are 2KiB.

Reply:
See https://github.com/filecoin-project/specs-actors/pull/363/files/217948049d2bf4c7a0ef660836d0ee9574911aa2#diff-2ecd9908929bba03f96f2c075d8d7ac5 for what we should be doing (in the test entry point)